### PR TITLE
Localise readme.md in hindi

### PR DESCRIPTION
**Pull request by mittalyashu**
This PR contains the changes to the `readme-hi.md` which is a translated document in the Hindi language.

Closes: https://github.com/kubernetes/website/issues/13909

### CODE REVIEW

**Comment by mittalyashu** [comment]
I have signed the CLA

**Comment by raelga** [comment]
Thank you very much for the Hindi localization @mittalyashu!

One of the [required files for starting the a new localziation](https://github.com/kubernetes/website/blob/master/content/en/docs/contribute/localization.md#minimum-required-content) is the Community Code of Conduct. Please open a PR in [cncf/foundation ](https://www.github.com/cncf/foundation) and submit the localized version of the [cncf/foundation /code-of-conduct.md](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Then, update the [README-hi.md](https://github.com/kubernetes/website/pull/13910/files#diff-211d06f59e6668cce0985743e83f1854) to point to the localized version and add it to the [/content/en/community](https://github.com/kubernetes/website/tree/master/content/en/community) folder of your localization.

**Comment by mittalyashu** [comment]
Sure.

**Comment by raelga** [comment]
You will also need to [join the Kubernetes organization](https://kubernetes.io/docs/contribute/localization/#join-the-kubernetes-github-organization) and [create the localization GitHub team](https://kubernetes.io/docs/contribute/localization/#add-your-localization-team-in-github).

**Comment by mittalyashu** [comment]
> Then, update the README-hi.md to point to the localized version

Where should I point to the localized version of COC file in `readme-hi.md`

> add it to the /content/en/community folder of your localization.

I didn't get this part @raelga

File: README-hi.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+[Community page](http://kubernetes.io/community/) पर कुबरनेट्स समुदाय के साथ जुड़ना सीखें।
+
+आप इस परियोजना के स्थानीयकरण तक पहुँच सकते हैं:
+
+- [Slack](https://kubernetes.slack.com/messages/sig-docs)
+- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
+
+## आचार संहिता
+
+कुबरनेट्स समुदाय में भागीदारी [कुबरनेट्स आचार संहिता द्वारा](code-of-conduct.md) शासित है.
```
Comment (raelga) [inline README-hi.md]: You can the link to the localized version of the Code of Conduct.

Comment (mittalyashu) [reply README-hi.md]: Sure.

Comment (mittalyashu) [reply README-hi.md]: I have updated the link to https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/hi.md

This file might not be accessible not, once the [PR](https://github.com/cncf/foundation/pull/39) is merged it should work.

File: README-hi.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# कुबरनेट्स प्रलेखन
```
Comment (nikhita) [inline README-hi.md]: This is probably me being paranoid.. :)

Hindi is written in the [Devanagri script](https://en.wikipedia.org/wiki/Devanagari)...could there be any problems with how this script is rendered with Hugo?

**Comment by nikhita** [comment]
As per https://kubernetes.io/docs/contribute/localization/#minimum-required-content, can you also add a new localization directory: https://kubernetes.io/docs/contribute/localization/#add-a-new-localization-directory? Thanks!

**Comment by raelga** [comment]
As @nikhita said the localized folder is required and you need at least one file there, otherwise Git will ignore the folder.

I just created a https://github.com/kubernetes/website/pull/13912 to add the OWNERS file setup to the [minimum-required-content](https://kubernetes.io/docs/contribute/localization/#minimum-required-content), completing that step will avoid the empty folder scenario.

File: README-hi.md (hunk 2 above)
Comment (mittalyashu) [reply README-hi.md]: I don't think there should be any problem with rendering, after all, it is just a text only.

**Comment by mittalyashu** [comment]
> As per https://kubernetes.io/docs/contribute/localization/#minimum-required-content, can you also add a new localization directory: https://kubernetes.io/docs/contribute/localization/#add-a-new-localization-directory? Thanks!

Will do that ASAP.

**Comment by mittalyashu** [comment]
@nikhita Is this the required content for `OWNERS` file?

```
# See the OWNERS docs at https://go.k8s.io/owners

# This is the localization project for French.
# Teams and members are visible at https://github.com/orgs/kubernetes/teams.

reviewers:
- sig-docs-hi-reviews

approvers:
- sig-docs-hi-owners

labels:
- language/hi
```

**Comment by raelga** [comment]
Update `French` with `Hindi`.

And the teams should be the same requested in the [Add your localization team in GitHub](https://kubernetes.io/docs/contribute/localization/#add-your-localization-team-in-github) section, but the names match the naming convention, so 👍 .

**Comment by nikhita** [comment]
@mittalyashu yup, except Hindi instead of French ;)

Also see https://github.com/kubernetes/website/pull/13912. You'll also need to update the OWNERS_ALIASES file at the root of this repo.

**Comment by mittalyashu** [comment]
**Question:** Can I add more translation to this PR or merge and close this PR just to keep it relevant with the issue.

**Comment by raelga** [comment]
The `_common-resources/images/*` are not needed for the localisation, if missing, will failback to the English version of the images.

**Comment by mittalyashu** [comment]
> The `_common-resources/images/*` are not needed for the localisation, if missing, will failback to the English version of the images.

Oh! Okay, will remove the images.

BTW, does it work for all the images or only in `_common-resources` directory.

**Comment by raelga** [comment]
@mittalyashu I'm Think that all images and pages failback to the Englsih version. So far, I only found that the code snippets from a file need to be inside the localized content folder.

About the PR, can you squash all your commits into one? Now the PR contains the creation and deletion of the images, among other commits.

Thanks!!

**Comment by mittalyashu** [comment]
> So far, I only found that the code snippets from a file need to be inside the localized content folder.

Okay

> About the PR, can you squash all your commits into one? Now the PR contains the creation and deletion of the images, among other commits.

But how? 😅

**Comment by raelga** [comment]
@mittalyashu Run `git rebase -i 0d46547655dbd21695c6d5311399048cb434e555~` to rebase to the first commit of this PR, and then mark all the commits but the first one as `squash`.

Check https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/ for more information.

If you want, we can do it together over Slack.

**Comment by Rajakavitha1** [comment]
@mittalyashu  Thanks for your contribution. I just made a few comments. Please take a look.

@daminisatya  could you please take a look as well.

**Comment by daminisatya** [comment]
Sure, I will take a look in the evening! Also, would be great if we have another reviewer!

@avidLearnerInProgress would be great to do peer reviews!

File: README-hi.md (hunk 2 above)
Comment (bhavin192) [inline README-hi.md]: Should this be कुबरनेटिस? Both the words Kubernetes and κυβερνήτης when pronounced sound more like कुबरनेटिस not कुबरनेट्स. I'm not sure though, but that's how I pronounce it.

**Comment by mittalyashu** [comment]
I have made the changes to the following comments @Rajakavitha1 

* https://github.com/kubernetes/website/pull/13910#pullrequestreview-229325238
* https://github.com/kubernetes/website/pull/13910#pullrequestreview-229339156
Only changed line number `10`.
* https://github.com/kubernetes/website/pull/13910#pullrequestreview-229340065
* https://github.com/kubernetes/website/pull/13910#pullrequestreview-229340928
* https://github.com/kubernetes/website/pull/13910#pullrequestreview-229341767
What you meant by rephrasing the words?
* https://github.com/kubernetes/website/pull/13910#pullrequestreview-229342635
* https://github.com/kubernetes/website/pull/13910#pullrequestreview-229343237

---

BTW, I am also unable to see any inline review comments from @Rajakavitha1 

CRNfMlC6sA

File: README-hi.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+कुबरनेट्स वेबसाइट को स्थानीय रूप से चलाने के लिए अनुशंसित तरीका एक विशेष [Docker](https://docker.com) image को चलाना है, जिसमें [Hugo](https://gohugo.io) स्टेटिक साइट जनरेटर शामिल है।
+
+> यदि आप विंडोज पर हैं, तो आपको कुछ और टूल्स की आवश्यकता होगी जिन्हें आप [Chocolatey](https://chocolatey.org) के साथ इंस्टॉल कर सकते हैं।
+
+> यदि आप Docker के बिना स्थानीय रूप से वेबसाइट चलाना पसंद करते हैं, तो नीचे Hugo का उपयोग करके स्थानीय रूप से साइट चलाना देखें।
+
+यदि आप Docker के बिना स्थानीय रूप से वेबसाइट चलाना पसंद करते हैं, तो नीचे दिए गए Hugo का उपयोग करके स्थानीय रूप से [साइट को चलाने](#running-the-site-locally-using-hugo) का तरीका देखें।
+
+यदि आप [डोकर](https://www.docker.com/get-started) चला रहे हैं, तो स्थानीय रूप से `कुबेरनेट्स-ह्यूगो` Docker image बनाएँ:
```
Comment (bhavin192) [inline README-hi.md]: Should we write it as डॉकर or डोकर? https://dictionary.cambridge.org/dictionary/english/docker US pronunciation sounds like  डॉकर but the I'm not able to decide if UK pronunciation sounds like डॉकर or  डोकर

File: README-hi.md (hunk 2 above)
Comment (mittalyashu) [reply README-hi.md]: For example, Contributor is written `योगदानकर्ता` but there another way too `योगदानकरता` which is not a good way of writing.

**Comment by bhavin192** [comment]
> BTW, I am also unable to see any inline review comments from @Rajakavitha1

I think that's because Rajakavitha1 added separate comments not inline comments

File: README-hi.md (hunk 3 above)
Comment (mittalyashu) [reply README-hi.md]: Now, this one is a bit tricky 😅

Comment (mittalyashu) [reply README-hi.md]: I did some digging and found that

### डोकर

The pronunciation of the word **डोकर** is `do-kar`

### डॉकर

The pronunciation of the word **डॉकर** is `doc-kr`

I know the pronunciation of both the word is almost the same but `डॉकर` is the correct word.

File: README-hi.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+## समुदाय, चर्चा, योगदान और समर्थन
+
+[Community page](http://kubernetes.io/community/) पर कुबरनेट्स समुदाय के साथ जुड़ना सीखें।
+
+आप इस परियोजना के स्थानीयकरण तक पहुँच सकते हैं:
+
+- [Slack](https://kubernetes.slack.com/messages/sig-docs)
+- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
+
+## आचार संहिता
```
Comment (mittalyashu) [inline README-hi.md]: Should I also change `आचार संहिता` to `कोड ओफ़ कंडक्ट`

Comment (Rajakavitha1) [reply README-hi.md]: yes, please

File: README-hi.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+कुबरनेट्स समुदाय में भागीदारी [कुबरनेट्स आचार संहिता द्वारा](https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/hi.md) शासित है.
+
+## धन्यवाद!
+
+कुबरनेट्स सामुदायिक भागीदारी पर पनपती है, और हम वास्तव में हमारी साइट और हमारे प्रलेखन में आपके योगदान की सराहना करते हैं!
```
[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply README-hi.md]: This is lot better!!!! I like it :)

File: README-hi.md (hunk 3 above)
Comment (Rajakavitha1) [reply README-hi.md]: Lets go with डॉकर!

File: README-hi.md (hunk 4 above)
Comment (mittalyashu) [reply README-hi.md]: Okay

File: README-hi.md (hunk 3 above)
Comment (mittalyashu) [reply README-hi.md]: Now we have found the official `डॉकर` word, should we change all Docker word to `डॉकर`?

**Comment by avidLearnerInProgress** [comment]
How do we go about links? Lets maintain a consistency throughout[Either in English or Hindi]

Example:
* [योगदान देना शुरू करें](https://kubernetes.io/docs/contribute/start/)
* [Staging Your Changes](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
* [पेज टेम्पलेट](http://kubernetes.io/docs/contribute/style/page-templates/)
* [प्रलेखन शैली गाइड](http://kubernetes.io/docs/contribute/style/style-guide/)
* [स्थानीयकरण कुबरनेट्स प्रलेखन](https://kubernetes.io/docs/contribute/localization/)

How about this:
_Staging your changes_ --> आपके परिवर्तनों का मंचन


@Rajakavitha1 @mittalyashu @daminisatya

**Comment by Rajakavitha1** [comment]
Hi

Yes, we need to maintain consistency for a better user experience.

I like what you suggested: आपके परिवर्तनों का मंचन


Others, please chime in your thoughts!!!!

Regards,
Rajie

**Comment by Rajakavitha1** [comment]
I prefer what you suggested : आपके परिवर्तनों का मंचन

Yes consistency is very important.

**Comment by Rajakavitha1** [comment]
@mittalyashu  this is an lgtm from me.
After at least one more reviewer gives an lgtm, I will mark it as approve.

/lgtm

**Comment by mittalyashu** [comment]
I totally agree with @avidLearnerInProgress, consistency is important but we cannot compromise with simplicity just to make it consistent, making the documentation easy to read and understand should be the first priority.

> How about this: [This is the closest I could come in Hindi]
> Staging your changes --> परिवर्तनों को करने के लिए इसे बारीक रूप से तैयार करें

@Rajakavitha1

**Comment by Rajakavitha1** [comment]
> परिवर्तनों को करने के लिए इसे बारीक रूप से तैयार करें

How about? आपके परिवर्तनों का अवलोकन करे

**Comment by avidLearnerInProgress** [comment]
> 
> 
> > परिवर्तनों को करने के लिए इसे बारीक रूप से तैयार करें
> 
> How about? आपके परिवर्तनों का अवलोकन करे because when you click on staging your changes it takes you to view your changes locally section.
> 
> @avidLearnerInProgress , @daminisatya please chime in.

This sounds great @Rajakavitha1 More crisp and simple.

**Comment by mittalyashu** [comment]
Great. Thank you for approving the PR.

**Comment by nikhita** [comment]
> Please feel free to assign me to any k/website PR for Hindi OWNERS updates.

/assign @zacharysarah 
Ref: https://github.com/kubernetes/org/pull/748#issuecomment-486448870

**Comment by zacharysarah** [comment]
@mittalyashu After a little bit of research, I figured out that the reason this PR has the `do-not-merge/invalid-owners-file` is due to another PR that has since merged: https://github.com/kubernetes/website/pull/14037/files

Please cherry-pick the relevant commit into this PR so we can proceed:
`git cherry-pick 997fe18`

Be sure to squash your commits after cherry-picking. 😄 

A change will remove the `LGTM` label, but any org member can `/lgtm` to add it again.

**Comment by Rajakavitha1** [comment]
@mittalyashu could you please address the changes that @zacharysarah has suggested!!!!!
I am hoping that we could have atleast one PR merged this week.

**Comment by mittalyashu** [comment]
> @mittalyashu could you please address the changes that @zacharysarah has suggested!!!!!
I am hoping that we could have at least one PR merged this week.

Yeah, sure.

**Comment by mittalyashu** [comment]
@zacharysarah, need urgent help.

https://kubernetes.slack.com/archives/DHZLPKD7D/p1556749734000200

Seems like I broke something 😱

**Comment by nikhita** [comment]
I'm working on fixing it.

**Comment by nikhita** [comment]
fyi, github is having some issues right now so this PR might not merge right away.

**Comment by nikhita** [comment]
Will leave lgtm to @zacharysarah or @Rajakavitha1.

**Pull request merged**